mmv1/third_party/terraform/website/docs/r/bigquery_table.html.markdown

```diff
@@ -446,7 +449,7 @@ exported:
 BigQuery tables imported using any of these accepted formats:
 
 ```
-$ terraform import google_bigquery_table.default projects/{{project}}/datasets/{{dataset_id}}/tables/{{table_id}}
-$ terraform import google_bigquery_table.default {{project}}/{{dataset_id}}/{{table_id}}
-$ terraform import google_bigquery_table.default {{dataset_id}}/{{table_id}}
+terraform import google_bigquery_table.default projects/{{project}}/datasets/{{dataset_id}}/tables/{{table_id}}
```

Comment:
Is it necessary to remove these`$`s? They are present on all of our other resource pages.